### PR TITLE
Create frenchConstants.tsx

### PR DESCRIPTION
**Pull request by lyonelf**

### CODE REVIEW

**Review by abhinavkgrd — DISMISSED** [review]

**Comment by abhinavkgrd** [comment]
recheck

**Comment by abhinavkgrd** [comment]
hey @lyonelf, so we did some changes to our CLA setup and seems like this branch was missed. 
Can you please resign the CLA, so that we can go ahead and get it merged 😄 .

and thank you again for your help !!

**Comment by lyonelf** [comment]
Hello,

I can't approuve CLA: 
2023-03-01_10h24_22

File: src/utils/strings/frenchConstants.tsx [hunk 1]
```diff
@@ -0,0 +1,918 @@
...
+    LOGIN: 'Connexion',
+    SIGN_UP: 'Inscription',
+    NEW_USER: 'Nouveau sur ente',
+    EXISTING_USER: 'Utilisateur existant',
+    NAME: 'Nom',
+    ENTER_NAME: 'Saisir un nom',
+    PUBLIC_UPLOADER_NAME_MESSAGE:
+        'Ajouter un nom afin que vos amis sachent qui remercier pour ces magnifiques photos!',
+    EMAIL: 'E-mail',
+    ENTER_EMAIL: 'Saisir l''adresse e-mail',
```
Comment (abhinavkgrd) [inline src/utils/strings/frenchConstants.tsx:52]: there are two single quotes after l, `l''adresse`
is it expected?
or should it have been `l'adresse` ?

Comment (abhinavkgrd) [reply src/utils/strings/frenchConstants.tsx:52]: Ahh okay, understood!
We are handling it a bit differently, I will update the code according.

**Review by abhinavkgrd — APPROVED** [review]

**Pull request merged**